doc/managing-wallet.md

```diff
@@ -0,0 +1,85 @@
+
+The following command, for example, creates a descriptor wallet:
+
+`$ bitcoin-cli -named createwallet wallet_name="wallet-01" descriptors=true`
+
+The `descriptors` parameter can be omitted if the intention is to create a legacy wallet.
+
+For now, the default type is the legacy wallet, but that should change in the near future.
+
+Wallets are created in the `wallets` folder of the data directory. The default folder varies by operating systems, as shown below. The user can also change the default by using the `-datadir` or `-walletdir` initialization parameters.
+
+| Operating System | Default wallet directory                                    |
+| -----------------|:------------------------------------------------------------|
+| Linux            | `home/<user>/.bitcoin/wallets`                              |
+| Windows          | `C:\Users\<user>\AppData\Roaming\Bitcoin\wallets`           |
+| macOS            | `/Users/<user>/Library/Application Support/Bitcoin/wallets` |
+
+### 1.2 Encrypting the Wallet
+
+The `wallet.dat` file is not encrypted by default and is, therefore, vulnerable if an attacker gains access to the device where the wallet or the backups are stored.
+
+The wallet may be encrypted with the following command:
+
+`$ bitcoin-cli -rpcwallet="wallet-01" encryptwallet "passphrase"`
+
+The `encryptwallet` command is used only when the wallet has not been encrypted yet. Otherwise, the `walletpassphrasechange` command should be used.
+
+`$ bitcoin-cli -rpcwallet="wallet-01" walletpassphrasechange "oldpassphrase" "newpassphrase"`
+
+The argument passed to `-rpcwallet` is the name of the wallet to be encrypted.
+
+The term "encrypt the wallet" used here is not very accurate. This command encrypts only the private key. All other wallet information, such as transactions, is still visible.
+
+The wallet can also be encrypted in the `createwallet` command via the `passphrase` argument:
+
+`$ bitcoin-cli -named createwallet wallet_name="wallet-01" descriptors=true passphrase="passphrase"`
+
+Note that if the passphrase is lost, all the coins in the wallet will also be lost forever.
+
+### 1.3 Backing Up the Wallet
+
+Wallets can be safely copied to another destination. This backup file should be stored on offline devices only, such as on a USB drive or an external hard drive.
+
+If the wallet and backup are lost for any reason, the bitcoins related to this wallet will become permanently inaccessible.
+
+The `bitcoin-cli backupwallet "destination"` RPC command or `Backup Wallet` GUI menu item must be used to ensure the file is in a safe state when the copy is made. The destination parameter must include the name of the file. Otherwise, the command will return an error message like "Error: Wallet backup failed!" for descriptor wallets. If it is a legacy wallet, it will be copied and a file will be created with the default file name `wallet.dat`.
```

Comment:
```suggestion
The `backupwallet` RPC or the `Backup Wallet` GUI menu item must be used to ensure the file is in a safe state when the copy is made. The destination parameter must include the name of the file. Otherwise, the command will return an error message like "Error: Wallet backup failed!" for descriptor wallets. If it is a legacy wallet, it will be copied and a file will be created with the default file name `wallet.dat`.
```